docs/rule/no-access-key.md

```diff
@@ -0,0 +1,22 @@
+## no-access-key
```

Comment:
Can you rename to `no-accesskey-attribute`? The attribute name is `accesskey` it seems odd to me to use alternate dasherization here just for that

Reply:
updated